### ISSUE TITLE
Accessibility: Toolbar

### ISSUE THREAD

**Issue by alansemenov** [enonic/lib-admin-ui#3626]
Toolbar is a `<div>`, hence we should implicitly communicate that it's a div element that serves as a toolbar. It should be focusable (currently it's not) and navigation inside toolbar (between toolbar's buttons) should be done with the arrow buttons, not with Tab.

- [x] `role="toolbar"`
- [x] `tabIndex="0"` on the toolbar
- [x] Remove `tabIndex` on ActionButtons
- [x] `aria-disabled="true"` on disabled buttons
- [ ] Implement navigation with arrows (moved to https://github.com/enonic/lib-admin-ui/issues/3642)
